README.md

```diff
@@ -1,36 +1,74 @@
-<div align='center'>
+# ![vkmod_smol](https://github.com/SpidFightFR/VulkanMod/assets/60798703/b3d31560-3563-4ddc-a9ea-d1fbf12b5b5e) VulkanMod
```

Comment:
@xCollateral The image is linked to my repo, what you can do is just copy it from the template i made, then paste it, it'll convert it into a link for your repo. :smile:

Reply:
Good idea 💡